src/tests/quantuminspire/qiskit/test_backend_qx.py

```diff
@@ -219,7 +282,7 @@ def test_get_experiment_results_returns_single_shot(self):
         api.get_jobs_from_project.return_value = [jobs]
         job = QIJob('backend', '42', api)
         simulator = QuantumInspireBackend(api, Mock())
-        experiment_result = simulator.get_experiment_results(job)[0]
+        experiment_result = simulator.get_experiment_results(job, False)[0]
```

Comment:
The meaning of the second argument is not clear. Consider using the explicit parameter name: `only_latest_run=False`

Reply:
Done